### DISCUSSION TITLE
Version 1.20.0

### DISCUSSION THREAD

**Discussion by apple502j** [ScratchAddons/ScratchAddons#3426] [Changelog]
## Bugfixes
- [#3365](https://github.com/ScratchAddons/ScratchAddons/issues/3365) message-filters hides studio promotion messages. (prio-3)
- [#3365](https://github.com/ScratchAddons/ScratchAddons/issues/3365) dark-www does not make transfer model dark. (prio-4)
- [#3365](https://github.com/ScratchAddons/ScratchAddons/issues/3365) scratchr2 does not apply to transfer modal. (prio-4)
- [#3365](https://github.com/ScratchAddons/ScratchAddons/issues/3365) studio-tools does not show Leave button after host transfer. (prio-3)
- [#3386](https://github.com/ScratchAddons/ScratchAddons/issues/3386) Some addons do not run in explore page popular mode. (prio-3)
- [#3387](https://github.com/ScratchAddons/ScratchAddons/issues/3387) better-featured-project is not compatible with user-id. (prio-4)
- [#3390](https://github.com/ScratchAddons/ScratchAddons/issues/3390) drag-drop shows "Move" tooltip on Windows. (prio-4)
- [#3403](https://github.com/ScratchAddons/ScratchAddons/issues/3403) block-switching uses wrong name for "start sound" block. (prio-4)
- [#3405](https://github.com/ScratchAddons/ScratchAddons/issues/3405) exact-count can show incorrect forum post counts. (prio-4)
- [#3432](https://github.com/ScratchAddons/ScratchAddons/issues/3432) variable-manager crashes when a specific extension is installed. (prio-4)
- [#3461](https://github.com/ScratchAddons/ScratchAddons/issues/3461) default-project causes uploaded projects to be replaced with the default project. (prio-3)
- [#3494](https://github.com/ScratchAddons/ScratchAddons/issues/3494) image-uploader does not upload dropped images on Firefox. (prio-3)
- [#3500](https://github.com/ScratchAddons/ScratchAddons/issues/3500) studio-tools crashes when promoting non-curator users. (prio-2)

## New Features
- [#3236](https://github.com/ScratchAddons/ScratchAddons/issues/3236) New addon: editor-unshare-button (prio-3)
- [#3299](https://github.com/ScratchAddons/ScratchAddons/issues/3299) New addon: editor-extra-keys (prio-3)
- [#3331](https://github.com/ScratchAddons/ScratchAddons/issues/3331) New addon: remix-button (prio-3)
- [#3365](https://github.com/ScratchAddons/ScratchAddons/issues/3365) scratch-messaging: support studio transfer notifications (prio-3)
- [#3365](https://github.com/ScratchAddons/ScratchAddons/issues/3365) scratch-notifier: notify studio transfer notifications (prio-3)
- [#3365](https://github.com/ScratchAddons/ScratchAddons/issues/3365) message-filters: support filtering studio transfer notifications (prio-3)
- [#3366](https://github.com/ScratchAddons/ScratchAddons/issues/3366) block-switching: support debugger blocks (prio-3)
- [#3409](https://github.com/ScratchAddons/ScratchAddons/issues/3409) editor-searchable-dropdowns: smarter searching (prio-3)
- [#3429](https://github.com/ScratchAddons/ScratchAddons/issues/3429) New addon: move-to-top-bottom (prio-3)
- [#3434](https://github.com/ScratchAddons/ScratchAddons/issues/3434) editor-dark-mode: add 2.0 preset (prio-3)
- [#3460](https://github.com/ScratchAddons/ScratchAddons/issues/3460) New addon: block-cherry-picking (prio-3)
- [#3465](https://github.com/ScratchAddons/ScratchAddons/issues/3465) New addon: hide-new-variables (prio-3)

**Comment by VeryUsual** [comment]
Yay!

**Reply by Samq64** [reply]
Next time it might be a good idea to just react with 👍 or 🎉 instead of posting "yay!" That's what reactions are for and they help keep the conversation more constructive.

**Reply by VeryUsual** [reply]
Ok

**Comment by CST1229** [comment]
The new features header is New Addon, even though there are other new features and more than 1 addon.

**Reply by Samq64** [reply]
See #3504.

> We won't have a video for v1.20.0

**Reply by CST1229** [reply]
I meant the post.
![kép](https://user-images.githubusercontent.com/68464103/133751675-ffbe5e80-0739-4aee-a56a-7a0c111d6508.png)

**Reply by Samq64** [reply]
Oops!
